docs/TempUG.md

```diff
@@ -330,78 +332,98 @@ Note you must provide a 'name' not already in the address book.
 
 
 #### For schedule list:
-<!-- use "SUBJECT" as parameter for lesson -->
-Format: `addLesson -name NAME [-day DATE] [-start TIME] [-end TIME] [-subject SUBJECT]` (for list specific format)
-* Format info 1
-* Format info 2
+
+Format: `addLesson -name NAME [-day DATE] [-start TIME] [-end TIME] [-subject SUBJECT]`
+* A new lesson cannot have the same name as existing students in the contact list.
+* Start time cannot be after end time.
```

Comment:
But it is ok to specifies only one of the start or end

Reply:
Make the necessary changes in the recent commit, thank you ! ◡̈